server/controllers/userAuth.js

```diff
@@ -24,8 +33,102 @@ const sendDemoUser = (req, resp) => {
   resp.redirect('/');
 };
 
+const signUp = (req, resp, next) => {
+  const { email, password, passwordConfirm, username } = req.body;
+  const sanitizedEmail = sanitize(email);
+  const sanitizedPassword = sanitize(password);
+  const sanitizedPasswordConfirm = sanitize(passwordConfirm);
```

Comment:
sanitizing is removing $ sing from strings, what if the user wants to have 'qwerty12345$' password?
Are we informing user on the front that $ is not allowed?